pennylane/templates/subroutines/adder.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
I think this is the correct copyright line
```suggestion
# Copyright 2024 Xanadu Quantum Technologies Inc.
```

Reply:
The original was correct.

Reply:
I was thinking of [this thread/particular message](https://xanaduhq.slack.com/archives/CA89H1BAN/p1717421936679049?thread_ts=1717420453.021679&cid=CA89H1BAN).

Reply:
Sounds good then.